src/BAP/SimpleBTSBundle/Form/EventListener/IssueSubscriber.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace BAP\SimpleBTSBundle\Form\EventListener;
+
+use BAP\SimpleBTSBundle\Entity\Issue;
+use BAP\SimpleBTSBundle\Entity\IssueResolution;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class IssueSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var EntityManager
+     */
+    protected $entityManager;
+
+    /**
+     * @param EntityManager $entityManager
+     */
+    public function __construct(EntityManager $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedEvents()
+    {
+        return [
+            FormEvents::PRE_SET_DATA => 'preSetData',
+            FormEvents::SUBMIT       => 'submit',
+        ];
+    }
+
+    /**
+     * @param FormEvent $event
+     */
+    public function preSetData(FormEvent $event)
+    {
+        $this->addIssueTypes($event);
+    }
+
+    /**
+     * @param FormEvent $event
+     */
+    public function submit(FormEvent $event)
+    {
+        /** @var Issue $issue */
+        $issue = $event->getData();
+
+        if ($issue && $issue->getResolution() === null) {
+            $defaultResolution = $this->entityManager
+                ->getRepository('BAPSimpleBTSBundle:IssueResolution')
+                ->findOneBy(['name' => IssueResolution::TYPE_UNRESOLVED])
```

Comment:
If your application will have translations for resolution - this code will not be workable.